src/tests/flyer.test.ts

```diff
@@ -202,4 +202,43 @@ describe('getTrending tests', () => {
     const getFlyersResponse = await FlyerRepo.getTrendingFlyers();
     expect(getFlyersResponse).toHaveLength(5);
   });
+
+  describe('getFlyersByCategorySlug tests', () => {
+    test('getFlyersByCategorySlug - no flyers', async () => {
```

Comment:
Let's update this test name to be more descriptive -- something like "invalid slug" should be contained in the name since we're testing how the function deals with invalid slugs